setup.py

```diff
@@ -85,7 +85,7 @@ def read(name):
     # needed to run some test pipelines
     'matplotlib',
     'seaborn',
-    'psycopg2',
+    'psycopg2-binary',
```

Comment:
why the change? psycopg2 works fine, the binary is only recommended when it isn't possible to install from source

Reply:
Does the psycopg2 works when developing? It caused a lot of issues that's why I changed it, but I can revert

Reply:
I also had issues with the `binary` I think it's because there aren't binaries for all platforms. I think let's leave the original one and add a suggestion to the CONTRIBUTING.md that they may also install the binary if this one has issues

Reply:
Yup makes sense